sites/fast-website/src/app/components/card-section/card-section.ts

```diff
@@ -0,0 +1 @@
+export class CardSection {}
```

Comment:
Should this extend anything or include anything? This class doesn't appear to support generic HTML attributes?

Reply:
It should.  Let me get that squared away.

Reply:
updated the index to extend FastElement.  Although as I think about it, that may not really be necessary.  Other than applying a filter on hover, this is all structural.  Sam and I are going to be looking at composing a more universal component for laying out the various cards on the page instead of having two different ones.